rosidl_generator_py/package.xml

```diff
@@ -36,8 +35,6 @@
   <test_depend>python3-pytest</test_depend>
   <test_depend>python_cmake_module</test_depend>
   <test_depend>rmw</test_depend>
-  <test_depend>rmw_implementation</test_depend>
```

Comment:
I believe this change led to a failure on the [binary job](http://build.ros2.org/job/Fbin_uF64__rosidl_generator_py__ubuntu_focal_amd64__binary/). We're generating messages for tests, but there's no C typesupport available (previously transitively brought in by `rmw_implementation`).

Here is a proposed fix: https://github.com/ros2/rosidl_python/pull/110